conda/activate.py

```diff
@@ -618,6 +618,38 @@ def _get_starting_path_list(self):
     def _get_path_dirs(self, prefix):
         if on_win:  # pragma: unix no cover
             yield prefix.rstrip("\\")
+
+            # We need to stat(2) for possible environments because
+            # tests can't be told where to look!
+            #
+            # mingw-w64 is a legacy variant used by m2w64-* packages
+            #
+            # We could include clang32 and mingw32 variants
+            variants = []
+            for variant in ["ucrt64", "clang64", "mingw64", "clangarm64"]:
+                path = self.sep.join((prefix, "Library", variant))
+
+                # MSYS2 /c/
+                # cygwin /cygdrive/c/
+                if re.match("^(/[A-Za-z]/|/cygdrive/[A-Za-z]/).*", prefix):
```

Comment:
will it only ever be a windows drive path? or could we also say:

```suggestion
                if "/" in prefix:
```

Reply:
@kenodegard to me that seems risky.  Can I create an expecting-to-be-Windows conda environment with `/` in the name?  Probably: `md here/there\conda-meta` (or whatever the minimum conda env is).
That said, anyone who mixes `/` and `\` in the same tree _and_ is using MSYS2 is probably in for a world of pain for other reasons.
I might think that `prefix.startswith("/")` would be OK but can you create an environment that _doesn't_ start at the filesystem root?  Can I `conda activate here\there` and what will the value of `prefix` be, will it always be resolved to the filesystem root?